### PR TITLE
Joy summarypagestyles

### PR DESCRIPTION
**Pull request by bestyoucanbe**
Updated with latest styles.  Pushed Will Smith's picture to the left and it is bigger.  Centered everything in the Flexbox section of this page.  Let me know if this is Okay with you!

### CODE REVIEW

**Comment by dhobson21** [comment]
Great job Joy! 👍

**Comment by joekennerly** [comment]
Commence the merge 👍

File: main.css [hunk 1]
```diff
@@ -54,6 +54,19 @@ main article img {
 }
 /* Joy's CSS Changes From Here */
 
+.willsmithimage { /* Make image a bigger size than the default */
+  width: 349px;
+}
+
+.articleonsumpage { /* Create a flexbox item on the main items on the page */
+  display: flex;
```
Comment (AdamKnowles) [inline main.css:93]: Nice

**Comment by AdamKnowles** [comment]
👍

File: index.html [hunk 2]
```diff
@@ -5,8 +5,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="main.css" />
-    <link rel="stylesheet" href="footer.css" />
-    <title>Document</title>
+    <link rel="stylesheet" href="footer.css">
+    <title>Will Smith</title>
```
Comment (joekennerly) [inline index.html]: sorry this caused you conflicts!

**Comment by bestyoucanbe** [comment]
> Great job Joy! 👍

Thanks!

**Comment by bestyoucanbe** [comment]
> Commence the merge 👍

Thanks!

**Comment by bestyoucanbe** [comment]
> 👍

Thanks Adam

**Pull request merged**